integration_tests/e2e/signIn.cy.ts

```diff
@@ -37,16 +36,6 @@ context('SignIn', () => {
     Page.verifyOnPage(AuthSignInPage)
   })
 
-  it('User can manage their details', () => {
```

Comment:
No longer relevant as this service is no longer serving a 'manage details' link.